docs/appendix/glossary.md

```diff
@@ -0,0 +1,53 @@
+# Glossary
+
+A glossary with explanations and background for wording used in the Glean project.
+
+## Glean
+
+According to the [dictionary](https://www.merriam-webster.com/dictionary/glean) the word “glean” means:
+
+> to gather information or material bit by bit
+
+Glean is the name of the whole product, see [Glean - product analytics & telemetry](https://docs.telemetry.mozilla.org/concepts/glean/glean.html).
+
+## Glean SDK
+
+The Glean SDK is the bundle of libraries with support for different platforms.
+The source code is available at <https://github.com/mozilla/glean>.
+
+## Glean SDK book
+
+This documentation.
+
+## Metric
+
+Metrics are the individual things being measured using Glean.
+They are defined in [metrics.yaml](https://mozilla.github.io/glean_parser/metrics-yaml.html) files.
+
+Glean itself provides some metrics out of the box.
+
+## Ping
+
+A ping is an entity used to bundle related metrics.
+The Glean SDK provides default pings and allows for custom ping, see [Glean Pings](../user/pings/index.md).
+
+## Pipeline
```

Comment:
```suggestion
## Glean Pipeline
```

Reply:
Since we're calling the SDK `Glean SDK` we should probably do the same with the pipeline.

Reply:
hm, the pipeline does ingest more than just Glean data at the moment though (and the linked article doesn't even mention Glean)

Reply:
True, but here we're really talking about the "Glean Pipeline", aren't we? That part of the pipeline has some Glean-specific moving parts (schema generator, transpiler, etc.).

Moreover, this ties into the "Glean" is "Glean SDK" + "Glean Pipeline" + "Glean tools"

Reply:
> Moreover, this ties into the "Glean" is "Glean SDK" + "Glean Pipeline" + "Glean tools"

Might be cool to add this statement the the `## Glean` section. But then with a new section about "Glean tools" and what they are.

Reply:
Note that that's already mentioned in the [product docs](https://docs.telemetry.mozilla.org/concepts/glean/glean.html).